docs/intro/reusable-apps.txt

```diff
@@ -288,6 +288,70 @@ working. We'll now fix this by installing our new ``django-polls`` package.
 
     pip uninstall django-polls
 
+Testing your app
+================
+Since we moved the ``polls`` directory out of the project,
+we don't have the project's ``settings.py`` and ``urls.py`` anymore.
+We need another way to configure our app in order to test it.
+
+#. First we create a ``tests`` directory and move the tests over from the
+   ``polls`` directory. Because the tests now reside in a different package
+   we will have to replace the relative import ``from .models import Poll``
+   with ``from polls.models import Poll``.
+
+#. Now we create a ``settings.py`` inside the ``tests`` folder which will contain
+   only the settings required to test our application.
+
+   .. code-block:: python
+       :caption: tests/settings.py
+
+       from django.urls import include, path
+
+       SECRET_KEY = 'fake-key'
+
+       INSTALLED_APPS = [
+           "polls",
+           # adding tests allows us to define models inside the tests package
+           # "tests",
+       ]
+
+       DATABASES = {
+           'default': {
+               'ENGINE': 'django.db.backends.sqlite3',
+               'NAME': ':memory:',
+           }
+       }
+
+       TEMPLATES = [
+           {
+               'BACKEND': 'django.template.backends.django.DjangoTemplates',
+               'APP_DIRS': True,
+           },
+       ]
+
+       ROOT_URLCONF = 'tests.urls'
+
+#. Because we test our views, we need a :setting:`ROOT_URLCONF` that includes
+   our app. We create an appropriate ``urls.py`` inside the ``tests``
+   directory.
+
+   .. code-block:: python
+       :caption: tests/urls.py
+
+       from django.urls import include, path
+
+       urlpatterns = [
+           path('polls/', include('polls.urls')),
+       ]
+
+#. Now we can run the tests. ::
+
+    python -m django test --settings=tests.settings tests
+
+For more information on testing reusable applications see :doc:`Advanced testing topics
+<../topics/testing/advanced>`.
```

Comment:
Would this be better linking to the `Using the Django test runner to test reusable applications` section directly?